diagnosis/clusterSanityCheck.sh

```diff
@@ -0,0 +1,230 @@
+#!/bin/bash
```

Comment:
All the other script names use lowercase only, this is only file name using camel case.
I prefer the term `health` (or similar) over `sanity`